k2/python/tests/ragged_ops_test.py

```diff
@@ -80,6 +82,52 @@ def test_to_list(self):
             self.assertEqual(k2.ragged.to_list(ans),
                              [[1, 2], [0], [3, 0], [2]])
 
+    def test_pad_ragged(self):
+        s = '''
+            [ [ 1 2 ] [ 3 ] [ ] [ 4 5 6 ] [ 7 8 9 10 ] ]
+        '''
+        for device in self.devices:
+            src = k2.RaggedInt(s).to(device)
+            padding_value = random.randint(0, 1000)
+            ans = k2.ragged.pad_ragged(src, padding_value)
+            expected = torch.ones((5, 4), dtype=torch.int32,
+                                  device=device) * padding_value
+            expected[0, 0] = 1
+            expected[0, 1] = 2
+            expected[1, 0] = 3
+            expected[3, 0] = 4
+            expected[3, 1] = 5
+            expected[3, 2] = 6
+            expected[4, 0] = 7
+            expected[4, 1] = 8
+            expected[4, 2] = 9
+            expected[4, 3] = 10
+
+            assert torch.all(torch.eq(ans, expected))
+
+    def test_pad_ragged_float(self):
+        s = '''
+            [ [ 1.0 2.0 ] [ 3.0 ] [ ] [ 4.0 5.0 6.0 ] [ 7.0 8.0 9.0 10.0 ] ]
+        '''
+        for device in self.devices:
+            src = k2.RaggedFloat(s).to(device)
+            padding_value = random.random() * 10
+            ans = k2.ragged.pad_ragged(src, padding_value)
+            expected = torch.ones((5, 4), dtype=torch.int32,
+                                  device=device) * padding_value
+            expected[0, 0] = 1.0
+            expected[0, 1] = 2.0
+            expected[1, 0] = 3.0
+            expected[3, 0] = 4.0
+            expected[3, 1] = 5.0
+            expected[3, 2] = 6.0
+            expected[4, 0] = 7.0
+            expected[4, 1] = 8.0
+            expected[4, 2] = 9.0
+            expected[4, 3] = 10.0
+
+            assert torch.all(torch.eq(ans, expected))
```

Comment:
```suggestion
            assert torch.allclose(ans, expected)
```
may be better for comparing floats.